test/test_ls_evolution_strategies.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
+import pytest
+
+import cgp
+from cgp.genome import ID_INPUT_NODE, ID_NON_CODING_GENE, ID_OUTPUT_NODE
+
+
+def test_step_towards_maximum(rng_seed):
+    primitives = (cgp.Parameter,)
+    genome = cgp.Genome(1, 2, 2, 1, None, primitives)
+    # [f0(x), f1(x)] = [c1, c2]
+    genome.dna = [
+        ID_INPUT_NODE,
+        ID_NON_CODING_GENE,
+        0,
+        0,
+        0,
+        0,
+        ID_OUTPUT_NODE,
+        1,
+        ID_OUTPUT_NODE,
+        2,
+    ]
+    ind = cgp.individual.IndividualSingleGenome(None, genome)
+    ind.idx = 0
+
+    def objective(f):
+        x_dummy = np.zeros((1, 1))  # input, not used
+        target_value = np.array([[1.0, 1.1]])
+        return -np.sum((f(x_dummy) - target_value) ** 2)
+
+    # test increase parameter value if too small
+    ind.genome._parameter_names_to_values["<p1>"] = 0.8
+    ind.genome._parameter_names_to_values["<p2>"] = 0.9
+    es = cgp.local_search.EvolutionStrategies(objective, rng_seed, max_generations=1)
+    es(ind)
+    assert ind.genome._parameter_names_to_values["<p1>"] > 0.8
+    assert ind.genome._parameter_names_to_values["<p2>"] > 0.9
+
+    # test decrease parameter value if too large
+    ind.genome._parameter_names_to_values["<p1>"] = 1.1
+    ind.genome._parameter_names_to_values["<p2>"] = 1.2
+    es = cgp.local_search.EvolutionStrategies(objective, rng_seed, max_generations=1)
+    es(ind)
+    assert ind.genome._parameter_names_to_values["<p1>"] < 1.1
+    assert ind.genome._parameter_names_to_values["<p2>"] < 1.2
+
+
+def _objective_convergence_to_maximum(f):
+    x_dummy = np.zeros((1, 1))  # input, not used
```

Comment:
why this name?

Reply:
no point, inputs are just irrelevant to the graph, so i thought to make this clear in the variable name. if this is confusing, i can change it?